src/dhcpv6.c

```diff
@@ -828,10 +828,10 @@ static void dhcpv6_send(enum dhcpv6_msg type, uint8_t trid[3], uint32_t ecs)
 	}
 
 	// Build IA_NAs
-	size_t ia_na_entries, ia_na_len = 0;
+	size_t ia_na_entry_count, ia_na_len = 0;
```

Comment:
In the `odhcpd` codebase, I've tried to standardise on `*_cnt` and `*_len`...don't know if that matters to you, but here you got a mix of both styles.

Reply:
It's undeniable that that style is unambiguous, though! XD

Reply:
I can't disagree with that :)